get-deps.sh

```diff
@@ -3,18 +3,20 @@
 set -e
 
 if [ "$GOPATH" = "" ]; then
-    export GOPATH=$(mktemp -d)
+    GOPATH=$(mktemp -d)
```

Comment:
Should we have quotes around this? I know its not (strictly) needed for $() but I wonder if we should still do it for consistently. OTOH it looks like we are a bit all over the place with $().